### ISSUE TITLE
Suppress warnings of "unmet peer dependencies" in frontend docker build

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#871]
### Terms

- [X] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

A general issue with the frontend Docker build that's come up is that we're consistently getting errors like the following when it's starting:

```bash
nuxt_frontend   | warning "nuxt > nitropack > @netlify/functions > @netlify/serverless-functions-api > @opentelemetry/otlp-transformer > @opentelemetry/core@1.23.0" has unmet peer dependency "@opentelemetry/api@>=1.0.0 <1.9.0".
```

The solution in the past has been to just install these dependencies, but if we keep on doing this then the `package.json` is going to get huge. It'd be great if we could look into options for running Nuxt within Docker that would allow us to ignore these warnings, and along with this we should then go through and find those dependencies that are not directly used in the frontend and remove them (assuming that they were installed to make these warnings go away).

Note - related issues are:
- #724
- #872

Happy to support on this or get to it myself! 😊

**Comment by andrewtavis** [comment]
CC @aasimsyed :) With the most recent yarn updates from the damn `yarn install` we have even more, so I think your suggestion of ignoring these should be acted on 😊 Just pinging you so you have an overview of this!

**Comment by andrewtavis** [comment]
Note that this should be worked on when we make all frontend dependencies explicit in #872 :)

**Comment by andrewtavis** [comment]
Closed by #951 :) I agree that this should be a problem anymore, @arescrimson! We can readdress it if it comes back in a new issue. Appreciate the support and you also pointing out how this one can also be closed!
